### ISSUE TITLE
Complete Scenarios

### ISSUE THREAD

**Issue by mjallday** [balanced/balanced-docs#451]
See discussion https://github.com/balanced/balanced-python/commit/37c33dfc3b5bc8de84bd5e9f37f2bd5b65a303c8#commitcomment-7749751

From @remear 

>  It is my opinion that the documentation still lacks a simple end-to-end example like https://github.com/balanced/balanced-python/blob/865be0b688be282fa84776becc9b631e222d4856/examples/examples.py. Such a topic in https://docs.balancedpayments.com/1.1/guides/quickstart/ would bring back that look-how-easy-this is feeling and the broad overview perspective of a full-cycle payment.

And me, @mjallday

> Examples of several operations together (e.g. tokenize a card, debit it, credit it to another funding instrument)

Can we create _runnable_ examples that show how to complete simple scenarios e.g. How to test a webhook, how to tokenize and credit a card, how to credit and debit an order, etc.

Scenarios I'd like to see covered:
- charging a card and paying out to a merchant
- charging multiple cards and paying out a single credit to a merchant (order)
- creating multiple orders and paying out a single credit to a merchant (settlement)
- partially refunding someone from a paid out order

**Comment by sophistry** [comment]
:+1:

**Comment by mjallday** [comment]
@rserna2010 pointed out that our client libraries having tests for some of these scenarios - example - https://github.com/balanced/balanced-php/blob/master/tests/Balanced/SuiteTest.php#L1195-L1223

this is good, we could look at linking to these tests from the documentation and then beefing up the comments on the tests to explain what is happening.

**Comment by sophistry** [comment]
Yes, the suite tests are helpful - but so is a nicely written description of the full 

debit-credit-account-settlement-reversal-refund-settle 

usage of the API showing common usage and answering questions
